sycl/test-e2e/DeviceCodeSplit/grf.cpp

```diff
@@ -13,7 +13,7 @@
 // - SYCL device binary images are compiled with the corresponding
 //   compiler option
 
-// REQUIRES: gpu && gpu-intel-pvc
+// REQUIRES: gpu && architecture-intel_gpu_pvc
```

Comment:
Why not just `REQUIRES: architecture-intel_gpu_pvc`

Reply:
> Why not just `REQUIRES: architecture-intel_gpu_pvc` ?

Because it was a mechanical find and replace :) But yes, we can now shorten some of those, `architecture-intel_gpu_pvc` implies `gpu`, of course.

Reply:
Yes. If it is not a lot of work, I'd also prefer to shorten/remove-redundant feature checks like this as a part of this PR.

Reply:
Done in 2ad997d3f1dcc37301b834b28225d1259cc11bce